Dalamud/Plugin/DalamudPluginInterface.cs

```diff
@@ -526,13 +526,40 @@ public void Dispose()
     /// <param name="affectedThisPlugin">If this plugin was affected by the change.</param>
     internal void NotifyActivePluginsChanged(PluginListInvalidationKind kind, bool affectedThisPlugin)
     {
-        this.ActivePluginsChanged?.Invoke(kind, affectedThisPlugin);
+        if (this.ActivePluginsChanged is { } callback)
```

Comment:
Would the extensions we already have work for this?
https://github.com/goatcorp/Dalamud/blob/5f62c703bff4137a1f887553fc1e0bd932d6dc6e/Dalamud/Utility/EventHandlerExtensions.cs#L22

Reply:
Nope, I tried to use them, but they don't work with the custom delegate we define in IClientState.